asteroid_definitions/solar-system.lua

```diff
@@ -14,19 +16,78 @@ local Uranus = require("asteroid_definitions.uranus")
 local Neptune = require("asteroid_definitions.neptune")
 local Pluto = require("asteroid_definitions.pluto")
 local AsteroidBeltOuter = require("asteroid_definitions.asteroid-belt-outer")
+local MakeMake = require("asteroid_definitions.makemake")
+local Eris = require("asteroid_definitions.eris")
+local Haumea = require("asteroid_definitions.haumea")
+local GongGong = require("asteroid_definitions.gonggong")
 local DefinitionsUtil = require("asteroid_definitions.definitions_util")
 -- Return all asteroid definitions for the solar system.
 
 -- Planets are sorted by the distance from the star they orbit.
 -- For instance, Earth is labelled under 3 - meaning that it's the third planet that orbits the Sun.
 -- Nil is currently representing planets which don't exist yet, such as the outer asteroid belt. These nils are defined in their respective asteroid definitions file.
-local Planets = {1 = Mercury, 2 = Venus, 3 = Earth, 4 = Mars, 5 = AsteroidBeltInner, 6 = Jupiter, 7 = Saturn, 8 = Uranus, 9 = Neptune, 10 = AsteroidBeltOuter}
+local Planets = {
+	[1] = Mercury,
+	[2] = Venus,
+	[3] = Earth,
+	[4] = Mars,
+	[5] = AsteroidBeltInner,
+	[6] = Jupiter,
+	[7] = Saturn,
+	[8] = Uranus,
+	[9] = Neptune,
+	[10] = AsteroidBeltOuter
+}
```

Comment:
Split planet definitions over multiple lines, allowing comments for each planet.